jest.js

```diff
@@ -94,9 +94,11 @@ async function run() {
         return;
     }
 
+    const current = process.cwd();
```

Comment:
I thought I reviewed this... don't we want to get the path from the `workingDirectory` arg above? I think it breaks in some cases otherwise....